web/Migrations/20220719095559_AddCharterAndLandlordStatusToLandlordAddHasRequestedAdminToUser.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace BricksAndHearts.Migrations
+{
+    public partial class AddCharterAndLandlordStatusToLandlordAddHasRequestedAdminToUser : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<bool>(
+                name: "HasRequestedAdmin",
+                table: "User",
+                type: "bit",
+                nullable: false,
+                defaultValue: false);
```

Comment:
This shouldn't be here.. make sure you're rebased onto main and try recreating the migration

Reply:
It looks like the existing migration 20220718155426_AddHasRequestedAdminToUser is deleted on this branch. Can you bring that back, then regenerate this migration, which then shouldn't be trying to add this column again